spec/system/private_message_map_spec.rb

```diff
@@ -39,6 +39,12 @@ def avatar_url(user, size)
   end
 
   it "updates the various topic stats, avatars" do
+    if ENV["CI"]
+      skip(
+        "This is flaky because it relies a lot on messagebus events and the counts don't always update in time",
+      )
+    end
```

Comment:
Should we add a `def skip_flaky(explanation)` or similar helper method for this?

Reply:
Haha you read my mind! I did think about that, I think I will do it now that someone else is entertaining the idea.